### DISCUSSION TITLE
如何在现有的k8s集群中安装milvus集群版？

### DISCUSSION THREAD

**Discussion by dushulin** [milvus-io/milvus#28525] [Q&A and General discussion]
背景：目前已有生产环境的k8s集群，没有设置default的storageclass，直接设置可能会影响其他pod
问题：
1. 集群中有juicefs，支持动态pv，是否可以直接使用该storageclass
2. 或者是否可以创建一个使用物理机存储的storageclass，然后再手动设置pv绑定到该storageclass呢？
3. 是否有在现有k8s中安装集群版的参数推荐，或最佳实践文档
PS：上述不使用default的storageclass的方式，是否可以直接通过helm的参数进行指定？

**Comment by LoveEachDay** [comment]
1. 可以
2. 可以, 这个会影响pod调度
3. 可以在helm chart 指定storageclass

**Reply by dushulin** [reply]
2. 我的表述可能有些误导，原意是我设置一个可以动态创建pv的storageclass，这个storageclass创建的具体存储是利用local的nvme，这样会影响pod调度吗？

**Reply by dushulin** [reply]
使用juicefs的sc，etcd会出现WAL的创建失败问题。
<img width="1769" alt="image" src="https://github.com/milvus-io/milvus/assets/25276301/5c145180-0197-4656-bce5-46a12a54baa8">
是必须使用本地盘吗？

**Comment by dushulin** [comment]
信件已收到      【自动回复】
